lib/rules/prefer-ember-test-helpers.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const emberUtils = require('../utils/ember');
+const { getImportIdentifier } = require('../utils/import');
+
+//-------------------------------------------------------------------------------------
+// Rule Definition
+//-------------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'enforce import of @ember/test-helpers method to ensure native method on `window` is not incorrectly used',
+      category: 'Testing',
+      recommended: false,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/prefer-ember-test-helpers.md',
+    },
+    schema: [],
+  },
+
+  create: (context) => {
+    if (!emberUtils.isTestFile(context.getFilename())) {
+      return {};
+    }
+
+    let hasImportedBlur = undefined;
+    let hasImportedFind = undefined;
+    let hasImportedFocus = undefined;
+
+    const showErrorMessage = (node, methodName) => {
+      context.report({
+        data: { methodName },
+        message: 'Import the `{{methodName}}()` method from @ember/test-helpers',
+        node,
+      });
+    };
+
+    return {
+      ImportDeclaration(node) {
+        hasImportedBlur = getImportIdentifier(node, '@ember/test-helpers', 'blur');
+        hasImportedFind = getImportIdentifier(node, '@ember/test-helpers', 'find');
+        hasImportedFocus = getImportIdentifier(node, '@ember/test-helpers', 'focus');
+      },
+      CallExpression(node) {
+        if (!hasImportedBlur) {
+          if (node.parent && node.parent.type === 'AwaitExpression') {
```

Comment:
I don't think we need the `await` check anymore now that we limited this rule to test files only right? I.e. any call to `focus` or similar functions in a test file should be imported from `@ember/test-helpers`. Seems too restrictive to me given that some developers might use `.then` in their tests instead of `await` and wouldn't be able to take advantage of this rule as a result.

Reply:
Ah yeah fair! I was thinking from the perspective of not checking unnecessarily, but removing the `AwaitExpression` check!